copilot/environments/overrides/cfn.patches.yml

```diff
@@ -29,13 +29,13 @@
   value: tcp
 - op: replace
   path: /Resources/EnvironmentHTTPSecurityGroupIngressFromPublicALB/Properties/Description
-  value: HTTP ingress from the public ALB on port 4000
+  value: HTTP ingress from the public ALB on port 80
 - op: add
   path: /Resources/EnvironmentHTTPSecurityGroupIngressFromPublicALB/Properties/FromPort
-  value: 4000
+  value: 80
 - op: add
   path: /Resources/EnvironmentHTTPSecurityGroupIngressFromPublicALB/Properties/ToPort
-  value: 4000
+  value: 80
```

Comment:
Do we know what we'll need to do to make these changes take? We'll need to be sure the load balancers forwarders are re-written, that may require an env deploy if not more. Worth testing.

Reply:
Not sure, I wasn't very confident making the changes to this file, one option could be to extract the change to use Thruster to a separate PR so we can at least switch to Rails 8.0 first.

Reply:
Tbh, I think we don't have to use port 80. Most operating systems require root to attach something to port 80 -- could make local dev harder for no reason, say if someone was using Docker to run the app. Let's stick to 4000 which works fine.

Reply:
We could switch to 80 in a separate PR if we think it's worth it.

Reply:
Maybe let's test a deploy from this branch to one of the envs?

Reply:
> Tbh, I think we don't have to use port 80. 

Yeah, this is another reasonable option. I switched to 80 to match the Rails defaults, but we could stick with 4000.

Reply:
Yeah, I had trouble in a separate project getting Fly to work on 80 so switched to 3000 which is the previous default. It's not a big difference.

The reason we use 4000 locally is to allow the prototype to run on 3000.

Reply:
I've configured Thruster to run on port 4000.